memory_w25q1128jv_spi/memory_w25q1128jv_spi.h

```diff
@@ -0,0 +1,186 @@
+/*
+ * W25Q128JV_MEMORY.h
+ *
+ *  Created on: Jan 1, 2024
+ *      Author: rober
+ */
+#pragma once
+
+#include <cmath>
+
+#if defined(STM32F1)
```

Comment:
let's verify compatibility on F4 series and add support

Reply:
good idea, will take it out until verified compatible.